app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, counter_cache: :comments_counter
+
+  def update_comment_counter
```

Comment:
- [ OPTIONAL ] It would be better if use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comment_counter` method every time a new comment has been saved to the database, then make the `update_comment_counter` method private. :+1: